### ISSUE TITLE
jsonld-context-autogen missing the base IRI

### ISSUE THREAD

**Issue by tekrajchhetri** [brain-bican/models#88]
@puja-trivedi @djarecka I noticed the context file is missing the default IRI. 

Issue:
Whenever you do the conversion of JSON-LD to ttl file, the file path is taken as base IRI, while it should be something like `http://example.org/NIMP/<ID>.`.
![Screenshot 2024-11-20 at 1 02 35 PM](https://github.com/user-attachments/assets/0d625b93-3335-4827-ba5f-76265006c89a)

In the previous version, it's not the case.
![Screenshot 2024-11-20 at 1 03 22 PM](https://github.com/user-attachments/assets/5d992bdd-4737-4135-a7eb-37e47d10aaac)

**Comment by puja-trivedi** [comment]
@tekrajchhetri - I took a look at the librayr_generation model and the NIMP prefix is already set to `http://example.org/NIMP/`. See line#18 -> https://github.com/brain-bican/models/blob/main/linkml-schema/library_generation.yaml.

While we did not explicitly define the prefix in the spreadsheet; linkml automatically fills in any missing prefix (that is why it is not missing from the .yaml file).

**Comment by tekrajchhetri** [comment]
@puja-trivedi Yes, there exist NIMP prefix but in the JSON-LD files that you've shared with me, they're missing the NIMP prefix as I've pointed out in the earlier example ([https://github.com/brain-bican/models/issues/88#issuecomment-2489311092](https://github.com/brain-bican/models/issues/88#issuecomment-2489311092)). The same is shown in the image below, which is a snapshot of a JSON-LD file. Also, you can see from the image blow that not in all cases NIMP prefix is missing. 

![image](https://github.com/user-attachments/assets/15dff552-70ac-4a2d-aa26-1fbc0c1a0944)

 It could be the issue with the translator that you're using to generate JSON-LD files. Also a suggestion, I think it would be better to add some automated checks to ensure there's always a prefix associated.
